rllib/policy/torch_policy.py

```diff
@@ -232,7 +232,7 @@ def learn_on_batch(self, postprocessed_batch):
         loss_out = force_list(
             self._loss(self, self.model, self.dist_class, train_batch))
         assert len(loss_out) == len(self._optimizers)
-        assert not any(np.isnan(l.detach().numpy()) for l in loss_out)
+        # assert not any(np.isnan(l.detach().numpy()) for l in loss_out)
```

Comment:
Remove instead of commenting out?

Reply:
+1
I would have liked to keep it in, but it breaks some dummy-agent test cases.